futurecollars/src/main/java/com/futurecollars/lekcja3/zadanie4/ConvertingMinutesToSeconds.java

```diff
@@ -0,0 +1,9 @@
+package com.futurecollars.lekcja3.zadanie4;
+
+public class ConvertingMinutesToSeconds {
+
+    public static int ConvertingMinutesToSeconds(int Minutes) {
+        int MinutesToSeconds = Minutes * 60;
+        return MinutesToSeconds;
+    }
```

Comment:
nie powinieneś używać do takich rzeczy konstruktora, niech będzie to zwykła metoda publiczna

Reply:
ta metoda nie powinna być też statyczna